movement-history.php

```diff
@@ -23,9 +19,9 @@
 
         foreach($movements as $m){
 
-            echo "<div style='background-color:lavender;padding:5px;margin:5px'>"; //opening div for each movement to have it's own little visual container
+            echo "<div class='movement-history-box'>"; //opening div for each movement to have it's own little visual container
 
-            echo "<div onclick='Show(".$m['movementId'].")' >↓".$m['movementName']."</div>".'<br>';
+            echo "<div style='font-size: 2.5vh;' onclick='Show(".$m['movementId'].")' >↓".$m['movementName']."</div>".'<br>';
```

Comment:
lol I know I said to use `vh` in an earlier comment, but it feels a bit strange to use that for font size. Is there a reason you're using this unit instead of `px` or `em`? If I'm not mistaken, this will change the font size based on the height of the browser which isn't normally what you want.

I noticed the same thing in a handful of places lower in this PR. If this is deliberate, no problem, you can keep it how it is, but it just stood out as something I've never seen before.

Reply:
Have you tried adjusting the height of the browser on desktop? I would expect the font to get really weird on a short desktop browser.

The two more common approaches I've seen are:

- Use `px` as your unit, and use CSS media queries to set different font sizes depending on the width (not height) of the browser. So you'd have a set of desktop font sizes and a set of mobile font sizes.
- Have a `font-size` set in pixels on the `<body>` tag, and then everything inside of it would use `em` (which is basically a % of the base font size on the body). Then, on mobile, you use a media query to change the base font size on the body tag and everything else would adjust automatically.